Tests/XcodeprojTests/PackageGraphTests.swift

```diff
@@ -137,7 +137,7 @@ class PackageGraphTests: XCTestCase {
                 XCTAssertEqual(targetResult.commonBuildSettings.OTHER_LDFLAGS?.first, "$(inherited)")
                 XCTAssertEqual(targetResult.commonBuildSettings.OTHER_SWIFT_FLAGS?.first, "$(inherited)")
                 XCTAssertEqual(targetResult.commonBuildSettings.SKIP_INSTALL, "YES")
-                XCTAssertEqual(targetResult.target.buildSettings.xcconfigFileRef?.path, "../Overrides.xcconfig")
+                XCTAssertEqual(targetResult.target.buildSettings.xcconfigFileRef?.path, RelativePath(unsafeUncheckedPath: "../Overrides.xcconfig").pathString)
```

Comment:
curios, why do we need to use the unsafeUncheckedPath variant?

Reply:
Because we cannot normalize `../Overrides.xcconfig` without an anchor.